app/lib/pdf/styles/base.rb

```diff
@@ -55,15 +55,45 @@ def freeze_style_hash(struct = styles)
       end
     end
 
-    private
+    class FontLiberation
+      FILE_REGULAR = "LiberationSans-Regular.ttf"
+      FILE_BOLD = "LiberationSans-Bold.ttf"
+      FAMILY = "Liberation Sans"
+
+      private_constant :FAMILY, :FILE_BOLD, :FILE_REGULAR
+
+      class << self
+        def family
+          FAMILY
+        end
 
-    def liberation_dir
-      %w[
-      /usr/share/fonts/liberation
-      /usr/share/fonts/liberation-sans
-      ].find do |path|
-        File.directory? path
+        def bold
+          @bold ||= where(FILE_BOLD)
+        end
+
+        def regular
+          @regular ||= where(FILE_REGULAR)
+        end
+
+        private
+
+        def where(file)
```

Comment:
IMO, the name `where` is a bit too abstract. I'd probably call it something like `file_path`

Reply:
updated to `#find_font`, is this better?

Reply:
I like it better than `where`, yes :)